stacks/jupyterhub-keycloak/jupyterhub-native-auth.yaml

```diff
@@ -4,7 +4,7 @@ name: jupyterhub
 repo:
   name: jupyterhub
   url: https://jupyterhub.github.io/helm-chart/
-version: 4.1.0 # 5.2.1
+version: 4.2.0 # 5.3.0
```

Comment:
Just a note: it is not *that* crucial that this helm is always bumped as a) the demo does not actual include and SDP elements (except stuff for S3 connectivity), b) care needs to be taken that the heml chart, the jupyterhub image, and any images used within the notebook, are in sync (same python version etc.)

Reply:
Added comment in 87e77d6.

Reply:
Thanks!